drivers/fuel_gauge/max17048/max17048.c

```diff
@@ -175,23 +175,24 @@ static int max17048_init(const struct device *dev)
 /**
  * Get a single property from the fuel gauge
  */
-static int max17048_get_single_prop_impl(const struct device *dev, struct fuel_gauge_property *prop)
+static int max17048_get_single_prop_impl(const struct device *dev, fuel_gauge_prop_t property_type,
+					 union fuel_gauge_prop_val *prop)
```

Comment:
Maybe use `prop` and `val` for `fuel_gauge_prop_t` and `union fuel_gauge_prop_val`?